src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {};
+
+  calculator.result = 0;
+
+  calculator.add = (number) => {
+    calculator.result += number;
+  };
+
+  calculator.subtract = (number) => {
+    calculator.result -= number;
+  };
+
+  calculator.multiply = (number) => {
+    calculator.result *= number;
+  };
+
+  calculator.divide = (number) => {
+    calculator.result /= number;
+  };
```

Comment:
[CODE KNOWLEDGE] - When creating methods in an object, you don't need to use the `function` keyword, use the method shorthand instead. For example, instead of `add: (number) => {...}`, you can simply write `add(number) {...},`.